Change unload NPC behavior to kill mobs that were spawned by unloaded NPC (non-permanent monster spawns) [Issue #2530] (#2590)
HerculesWS/Hercules: doc/atcommands.txt

---------------------------------------

@unloadnpc <NPC_name> {<flag>}

Unloads a NPC.
Flag:
	0 - do not unload mobs spawned by NPCs in file

> 	1 - unload mobs spawned by NPCs in file

Reviewer: maybe mention here too about commands?
```suggestion
	1 - unload mobs spawned by NPCs and atcommands
```

Author: In my opinion this would be confusing, because it's really just _atcommand(@monster ...)_ used inside a script. Atcommands used by players are not affected.
